p2panda-rs/src/operation/validate.rs

```diff
@@ -117,7 +117,38 @@ pub fn validate_operation_with_entry(
     })
 }
 
-/// Checks the fields of an operation-like data type against a schema.
+/// Check the format of an operation-like data type.
+///
+/// This method checks against:
+///
+/// 1. Correct operation format (#OP2)
+pub fn validate_operation_format<O: Actionable + Schematic>(
+    operation: &O,
+) -> Result<(), ValidateOperationError> {
+    let action = operation.action();
+    let fields = operation.fields();
+    let previous_operations = operation.previous_operations();
+
+    match (action, fields, previous_operations) {
```

Comment:
This is a little hard to read, the implementation before was maybe more verbose but easier to follow .. do you mind writing it like that?